### ISSUE TITLE
Slow Learning and Overfitting - CNN

### ISSUE THREAD

**Issue by HussamCheema** [keras-team/keras#11692]
My CNN model learning very slow after some epochs and testing accuracy is not increasing. I'm using **learning rate = 0.0003** and **dropout = 0.55**. **Batch Size = 50**, **Training Samples = 8000** grayscale images and **Testing Samples = 1600** grayscale images. The model is given below. 

```
model = Sequential()
model.add(Conv2D(kernel_size=(3,3),filters=12,input_shape=(64, 64, 1),activation="relu",padding="valid"))
model.add(Conv2D(kernel_size=(3,3),filters=10,activation="relu",padding="same"))
model.add(MaxPooling2D(pool_size=(2,2),strides=(1,1)))
model.add(Conv2D(kernel_size=(3,3),filters=4,activation="relu",padding="same"))
model.add(Conv2D(kernel_size=(2,2),filters=5,activation="relu",padding="same"))
model.add(MaxPooling2D(pool_size=(3,3),strides=(1,1)))
model.add(Conv2D(kernel_size=(3,3),filters=5,activation="relu",padding="same"))
model.add(Flatten())
model.add(Dropout(0.55))
model.add(Dense(100,activation="sigmoid"))
model.add(Dense(1,activation='sigmoid'))
model.summary()

model.compile(loss=keras.losses.mean_squared_error,
              optimizer=keras.optimizers.Adam(lr=0.0003),
              metrics=['accuracy'])
```

![image](https://user-images.githubusercontent.com/17239812/48775610-61fb5400-ecef-11e8-9cc3-57ee1df8f657.png)

How do I increase training and testing accuracy?

![image](https://user-images.githubusercontent.com/17239812/48775842-0aa9b380-ecf0-11e8-908a-3f6df616d0e4.png)

This is the accuracy and loss I'm getting after 30th epoch. Training accuracy is on 0.77 for last 5 epochs!!

**Comment by HussamCheema** [comment]
![image](https://user-images.githubusercontent.com/17239812/48775760-d33b0700-ecef-11e8-85bf-23ba6198f04a.png)
This is the accuracy and loss I'm getting after 30th epoch.
